traffic-policy/gallery.mdx

```diff
@@ -3,7 +3,7 @@ import ConfigExample from "/src/components/ConfigExample.tsx";
 export const Deny = () => (
 	<ConfigExample
 		config={{
-			inbound: [
+			on_http_connect: [
```

Comment:
who's she?

jk did we want on_tcp_connect or on_http_request here and below?

Reply:
Yeah, would be nice to actually have multiple examples showcasing different use-cases now. Denying traffic instantly for IPs through the on_tcp_connect phase, or blocking traffic from specific users on the HTTP request phase.

Reply:
should be `on_http_request` in this specific one